app/build.gradle

```diff
@@ -57,6 +57,10 @@ dependencies {
       'androidx.core:core-ktx:1.0.2',
       'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-alpha03',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
+      'androidx.cardview:cardview:1.0.0',
```

Comment:
@BenHenning Will need suggestions here.

For testing purposes @veena14cs is accessing `data` module directly from `app` module and based on App Architecture Document, I think that we cannot access `data` module directly from `app` module. It should be accessed via `domain` module.

Reply:
Hi @veena14cs -- just a note for the future: before marking a comment as resolved, please be sure to explain in a reply to the comment thread what the resolution is, for future reference.

Thanks!

Reply:
Sure I will do that.

Reply:
Above dependency is been removed as it was added for debugging purpose